### DISCUSSION TITLE
How to remove the logo from the title slide?

### DISCUSSION THREAD

**Discussion by rbcavanaugh** [quarto-dev/quarto-cli#11288] [Q&A]
### Description

I see that there is a PR for this (#9558) but I'm not exactly sure how to implement this for the title slide. I have a logo specified in the YAML header, but since I have a background image on the title slide, I'd like to hide the logo. 

thanks!

**Comment by mcanouil** [comment]
See other discussions:

- https://github.com/quarto-dev/quarto-cli/discussions/557
